utility/src/main/java/org/oppia/android/util/parser/OppiaImage.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.android.util.parser
```

Comment:
Should be in the same package as ``ImageLoader``.

Reply:
I guess it is already or We need to make a separate package for imageLoader and OppiaImage ?

Reply:
Hmm. I misread the ImageLoader package. They should be in a different package, but let's not fix that now. This is fine as-is.